pennylane/interfaces/jax.py

```diff
@@ -381,9 +381,7 @@ def execute_new(tapes, device, execute_fn, gradient_fn, gradient_kwargs, _n=1, m
 
     Args:
         tapes (Sequence[.QuantumTape]): batch of tapes to execute
-        device (.Device): Device to use to execute the batch of tapes.
-            If the device does not provide a ``batch_execute`` method,
-            by default the tapes will be executed in serial.
+        device (.Device): Device to use for the shots vectors.
```

Comment:
Is this object always passed in the shot vector case now?

Reply:
Yes, devices were always passed before the return types project. I removed it for the Jax integration PR but I had to add it back for the shot vectors.